datasets/gov/pagd/evakuacijos_punktas.csv

```diff
@@ -0,0 +1,32 @@
+id,dataset,resource,base,model,property,type,ref,source,prepare,level,access,uri,title,description
+,datasets/gov/pagd/evakuacijos_punktai,,,,,,,,,,,,pagd evakuacijos punktai,"""Gyventojų surinkimo punktai"""
+,,,,,,prefix,spinta,,,,,https://github.com/atviriduomenys/spinta/issues/,,
+,,,,,,,manifest,,,,,https://github.com/atviriduomenys/manifest/issues/,,
+,,,,,,,,,,,,,,
+,,,,Ecakuacijos_punktas,,,vda_id,,,4,,,,"""Gyventojų surinkimo punktai – tai vietos, į kurias turės susirinkti gyventojai."""
```

Comment:
Modelio pavadinimas be skyrybos ženklų turėtų būti, skirtingus žodžius atskiriame rašydami iš didžiosios raidės - šiuo atveju EvakuacijosPunktas.